packages/admin-ui-docs/docs/templates/listing page/overview.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Overview
+path: /listing-page/overview/
+sidebar_position: 1
+---
+
+import { Columns, Column, Box, IconArrowUpRight, Text, Flex, FlexSpacer } from '@vtex/admin-ui'
+
+Merchants use Listing pages to scan, sort, compare and take action on items of a specific object type, such as products, or record type, such as orders.
+
+<br />
+
+<Flex
+  align="stretch"
+  direction="column"
+  justify="space-around"
+  csx={{ border: '$neutral', borderRadius: 'default', overflow: 'hidden'}}
+>
+  <Box csx={{ borderRadius: 'inherit'}} >
+      <Box csx={{ borderRadius: 'inherit'}}><a href="#"><img src="https://user-images.githubusercontent.com/77292838/191526389-e0a97572-e01b-422a-8a96-56410d1f1e68.png"></img></a></Box>
+  </Box>
+  <Box csx={{ padding: '20px' }}>
+    <Flex
+    align="stretch"
+    direction="row"
+    justify="flex-end"
+    >
+      <a href="#" csx={{ float: 'right'}}>Demo<IconArrowUpRight csx={{ float: 'right'}} /></a>
+    </Flex>
+  </Box>
+</Flex>
```

Comment:
Is the `href` here with the right value?

Reply:
This link was pending before, but it's supposed to go to https://uwyp5t.csb.app/ (preview of the template on codesandbox).